store/db.go

```diff
@@ -33,6 +34,9 @@ type DB struct {
 	dbdir string
 	lock  *lock.FileLock
 	sqldb *sql.DB
+	// In fact, the mutex here is not necessary, as we already
+	// use flocks. This is just for making race detector happy.
+	sync.Mutex
 }
```

Comment:
Nits:
a) list the mutex last in the struct (by convention it's listed before things it protects, and it doesn't really protect lock.FileLock
b) change comment:
```
// Although the FileLock already ensures thread safety, the Go runtime is unaware 
// of this, and so Mutex is necessary to satisfy the Go race detector
```